docs/rfcs/0006-site-wide-notification.md

```diff
@@ -0,0 +1,48 @@
+## Site Wide Notifications
+
+#### Abstract
+
+As a member of the marketing team, I'd like to be able to post a message that will appear on all pages of the site, for example to advertise a promotion.
+
+#### Requirements
+
+- Brief description of the plan for adding, editing and removing the site-wide notification (see options below)
+- CMS user can add, edit and remove banner message through Wagtail. Or, if that is difficult, admin user can add, edit and remove banner message through the django admin
+- User can dismiss the banner by clicking on the X. It should stay hidden for the rest of the session
+
+##### Designs and Mockups
+- See, for example, [InVision](https://impactbnd.invisionapp.com/share/4TQEESTEYMZ#/screens/345573039)
+- This element should also be in the [course PSD](https://drive.google.com/file/d/10uAQ3emiF3ufd5eigYow0kxwA_51WiVD/view?usp=sharing)
+
+#### Architecture Changes
+
+##### Backend changes
+
+- Add `NotificationPage` in wagtail with following field
```

Comment:
I don't think this should be a page since it is content that appears on all pages, a snippet seems a better fit: http://docs.wagtail.io/en/v2.0/topics/snippets.html

Reply:
@rhysyngsun  just a note , I *think* pages would give us some features for free , like published/unpublished , or scheduled publishing http://docs.wagtail.io/en/v2.5.1/reference/pages/theory.html#scheduled-publishing

Reply:
Pages are typically associated with a url or a url scheme though, so it's bending wagtail to do something it wasn't designed for

Reply:
ok